ydb/docs/en/core/concepts/topic.md

```diff
@@ -102,7 +102,7 @@ To accurately calculate the balance, the message processing order is crucial. If
 
 When several application instances read messages from a stream, a message about account top-ups can be received by one instance and a message about debiting by another. In this case, there's no guaranteed instance with accurate balance information. To avoid this issue, you can, for example, save data in the DBMS, share information between application instances, and implement a distributed cache.
 
-{{ ydb-short-name }} can write data so that messages from one source (for example, about transactions from one account) arrive at the same application instance. The source of a message is identified by the source_id, while the sequence number of a message from the source is used to ensure there are no duplicate messages. {{ydb-short-name}} arranges data streams so that messages from the same source arrive at the same partition. As a result, transaction messages for a given account will always arrive at the same partition and be processed by the application instance linked to this partition. Each of the instances processes its own subset of partitions and there's no need to synchronize the instances.
+{{ ydb-short-name }} can write data so that messages from one source (for example, about transactions from one account) arrive at the same application instance. Each source writes messages with its own unique producer_id, and a sequence number (seqno) is used to prevent duplicate processing. {{ ydb-short-name }} routes all messages with the same producer_id to the same partition. As a result, transaction messages for a given account always arrive at the same partition and are processed by the application instance associated with it. Each instance handles its own subset of partitions, so there is no need for synchronization between instances.
```

Comment:
```suggestion
{{ ydb-short-name }} can write data so that messages from a single source are delivered to the same application instance. Each source writes messages with its own unique `producer_id`, and a sequence number (`seqno`) is used to prevent duplicate processing. {{ ydb-short-name }} routes all messages with the same `producer_id` to the same partition. Each instance handles its own subset of partitions, eliminating the need for synchronization between instances. For example, this approach could allow all transactions from a given account to be processed by the application instance associated with it. 
```